products/ASC.Files/Client/src/components/panels/SharingPanel/index.js

```diff
@@ -33,14 +33,15 @@ const i18n = createI18N({
   page: "SharingPanel",
   localesPath: "panels/SharingPanel",
 });
-
 const { changeLanguage } = commonUtils;
 const { ShareAccessRights } = constants;
 const {
   getCurrentUserId,
   getSettingsCustomNamesGroupsCaption,
 } = store.auth.selectors;
 
+const SharingBodyStyle = { height: `calc(100vh - 156px)` };
```

Comment:
Is inline style really necessary?

Reply:
This style is required for a custom scrollbar. Need inline, or !important